src/main/java/cart/controller/dto/ProductRequest.java

```diff
@@ -0,0 +1,20 @@
+package cart.controller.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import javax.validation.constraints.NotBlank;
+
+@Getter
+@AllArgsConstructor
+public class ProductRequest {
+
+    @NotBlank
+    private String name;
```

Comment:
검증에 통과하지 못했을 때 예외 메시지를 정해줄 수 있습니다 :)

```suggestion
    @NotBlank(message = "상품 이름은 필수 값 입니다.")
    private String name;
```

Reply:
해당 검증에 통과하지 못하면 어떤 예외가 발생하고 있을까요?

ExceptionHandler 에서 처리해볼까요 😄